metadata_tools/hosts/GO_0xxx/gcp_index_startup.sh

```
@@ -0,0 +1,22 @@
+apt-get update -y
+# Reading package lists... Done
+# E: Could not open lock file /var/lib/apt/lists/lock - open (13: Permission denied)
+# E: Unable to lock directory /var/lib/apt/lists/
+
+
+apt-get install -y python3 python3-pip python3-venv git
+cd
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)